test/models/project_media_8_test.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../test_helper'
+
+class ProjectMedia8Test < ActiveSupport::TestCase
+  def setup
+    super
```

Comment:
Tests will run faster if you don't run everything that `super` runs. I suggest you start with empty `setup` and `teardown` methods and then add only the steps you actually need:

```
def setup
end

def teardown
end
```